lib/AST/CMakeLists.txt

```diff
@@ -73,6 +77,7 @@ add_swift_library(swiftAST STATIC
     clangLex
     clangAPINotes
     clangBasic
+    ${mincore}
```

Comment:
I'm confused too - this seems to be the only way I avoid these linker errors. 
I genuinely don't know, sorry!